### PR TITLE
v0 of find_minimal_coverage

### PR DESCRIPTION
**Pull request by fcollman**
not completed yet complete, but scaffold of endpoint to find minimal set of nodes that covers all passed in nodes

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -11,7 +11,7 @@ flask
 flask_cors
 python-json-logger
 redis
-rq
+rq<2
```
Comment (fcollman) [inline requirements.in:14]: I noticed this was a problem when trying to get a local version of PCG running and so i added it here pre-emptively to save us time in the future when we try to recompile reqs.

File: pychunkedgraph/app/segmentation/common.py [hunk 2]
```diff
@@ -148,6 +148,91 @@ def handle_root(table_id, atomic_id):
...
+        min_layer, cg.meta.layer_count
+    ):  # Process from higher layers to lower layers
+        if len(node_queue[layer]) == 0:
+            continue
+        
+        current_nodes = list(node_queue[layer])
+
+        # Call handle_roots to find parents
+        parents = cg.get_roots(current_nodes, stop_layer=layer + 1, time_stamp=timestamp)
+        print(parents)
```
Comment (akhileshh) [inline pychunkedgraph/app/segmentation/common.py:197]: print statement

**Review by akhileshh — APPROVED** [review]
looks good except for the print statement

File: requirements.in (hunk 1 above)
Comment (akhileshh) [reply requirements.in:14]: I think this is fine for now, version 2 was released only recently in Oct.

**Pull request merged**